dace/codegen/targets/rtl.py

```diff
@@ -0,0 +1,559 @@
+# Copyright 2019-2020 ETH Zurich and the DaCe authors. All rights reserved.
+
+import itertools
+
+from typing import List, Tuple, Dict
+
+from dace import dtypes, config, registry, symbolic, nodes, sdfg
+from dace.sdfg import graph, state, find_input_arraynode, find_output_arraynode
+from dace.codegen import codeobject, dispatcher, prettycode
+from dace.codegen.targets import target, framecode
+from dace.codegen.targets.common import sym2cpp
+
+
+@registry.autoregister_params(name='rtl')
+class RTLCodeGen(target.TargetCodeGenerator):
+    """ RTL Code Generator (SystemVerilog) """
+
+    title = 'RTL'
+    target_name = 'rtl'
+    languages = [dtypes.Language.SystemVerilog]
+
+    def __init__(self, frame_codegen: framecode.DaCeCodeGenerator,
+                 sdfg: sdfg.SDFG):
+        # store reference to sdfg
+        self.sdfg: sdfg.SDFG = sdfg
+        # store reference to frame code generator
+        self.frame: framecode.DaCeCodeGenerator = frame_codegen
+        # add debug constant if not existent
+        if "DACE_VERILATOR_ENABLE_DEBUG" not in sdfg.constants:
+            sdfg.add_constant("DACE_VERILATOR_ENABLE_DEBUG", False)
```

Comment:
Like Tal said, I think we should only emit the debugging code if the debugging flag is set, and have the debugging flag as an option in the DaCe config. But maybe we leave this for a future PR, since it's not critical, and it would be good to get this merged :-)

Reply:
Yes, you are right, this is semi-properly solved. Since is it part of the dace.conf extension, i rather prefer to re-wire all the signals there at once.